components/DiscordBar.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react'
+import styles from '../scss/discordBar.module.scss'
+type Member = {
+  id: string
+  username: string
+  status: string
+  avatar_url: string
+}
+interface discordJSON {
+  presence_count: number
+  members: Member[]
+}
+const ONE_MINUTE = 60 * 1000
+const Member: React.FC<{ avatar: string }> = ({ avatar }) => {
+  return (
+    <div className="d-inline-block">
+      <img src={avatar} className={`${styles['member-avatar']} mx-1`} />
+    </div>
+  )
+}
+const DiscordBar: React.FC = () => {
+  const [data, setData] = useState<discordJSON>({
+    presence_count: 1,
+    members: []
+  })
+  const getData = async () => {
+    try {
+      let data = await fetch(
+        'https://discord.com/api/guilds/828783458469675019/widget.json'
+      )
+      data = await data.json()
+      setData((data as unknown) as discordJSON)
+      setTimeout(getData, ONE_MINUTE)
+    } catch (e) {
+      setTimeout(getData, ONE_MINUTE)
+    }
```

Comment:
Use `finally` so you don't have to repeat the timeout.

```suggestion
    } catch (e) {
      // maybe show some error message in this case?
    } finally {
      setTimeout(getData, ONE_MINUTE)
    }
```

Reply:
I added error state. See [new storybook](https://deploy-preview-645--youthful-thompson-448332.netlify.app/?path=/story/components-discord--error).